src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  selectedTodo?: Todo | null;
+  handleShowModal: (todo: Todo) => void;
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  selectedTodo,
+  handleShowModal,
+}) => (
+  <tr
+    data-cy="todo"
+    className={classNames({
+      'has-background-info-light': selectedTodo?.id === todo.id,
```

Comment:
DRY. Create variable for this value `selectedTodo?.id === todo.id` and reuse it here and in other places